Commands/Commands/queue.js

```diff
@@ -271,7 +271,7 @@ commands.registerVote = {
                     var reportsArr = msg.fetchReactions({
                       id: '302137374920671233',
                       name: 'report'
-                    }, bot.User)
```

Comment:
`bot.User` would be the correct usage, refer to the [discordie documentation.](qeled.github.io/discordie/)